src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -925,16 +934,17 @@ public String getCallSettingsTypeName(Interface apiInterface) {
   }
 
   /** The name of the return type of the given grpc streaming method. */
-  public String getGrpcStreamingApiReturnTypeName(Method method, ImportTypeTable typeTable) {
+  public String getGrpcStreamingApiReturnTypeName(
+      MethodContext methodContext, ImportTypeTable typeTable) {
```

Comment:
Why keep the context out of the namer?

A lot of the methods require the method, typetable, and methodconfig, and it just seemed a little verbose to use all of those parameters instead of a singular methodContext.